.vscode/launch.json

```diff
@@ -1,47 +0,0 @@
-{
```

Comment:
Why revome the VScode config? It's not unreasonable for a project to include how it should work with dev tools.

Reply:
Seems to be a developer-specific thing, i.e. I don't know if that should be part of puppetwash repo. If you think it still should be, then I can add it back.

Reply:
Not too picky. Sometimes people want to enforce a certain style.

Reply:
Yeah, I lean towards removing it. I don't think puppetwash is big enough to include that stuff (yet).